pkg/server/options/options.go

```diff
@@ -131,6 +131,11 @@ func NewOptions(rootDir string) *Options {
 	// turn on the watch cache
 	o.GenericControlPlane.Etcd.EnableWatchCache = true
 
+	// TODO(gman0): remove this admissionregistration.k8s.io/v1alpha1 override once
```

Comment:
Can we get issue created in kcp repo with links to code places where this is needs work ?

Reply:
Here it is: https://github.com/kcp-dev/kcp/issues/3303

Reply:
I'll open more for the rest of my TODOs.